lib/modules/manager/bundler/extract.spec.ts

```diff
@@ -141,4 +148,33 @@ describe('modules/manager/bundler/extract', () => {
       { depName: 'sfn_my_dep2', currentValue: '"~> 1"' },
     ]);
   });
+
+  it('parses git refs in Gemfile', async () => {
+    fs.readLocalFile.mockResolvedValueOnce(gitRefGemfile);
+    const res = await extractPackageFile(gitRefGemfile, 'Gemfile');
+    expect(res?.deps).toHaveLength(3);
+    expect(res?.deps).toMatchObject([
```

Comment:
```suggestion
    expect(res).toMatchObject({ deps: [
```

`toMatchObject` validates array length